test/_external/test262/skiplist

```diff
@@ -0,0 +1,125 @@
+# NEEDS INVESTIGATION!
+test/language/module-code/dynamic-import/eval-rqstd-once.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/dynamic-import/eval-self-once.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/eval-export-dflt-expr-err-eval.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/eval-gtbndng-indirect-update-dflt.js
+
+# export * from ns
+test/language/module-code/eval-rqstd-once.js
+
+# export * from ns
+test/language/module-code/eval-rqstd-order.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/eval-self-abrupt.js
+
```

Comment:
👆 is using `throw new Test262Error`

Reply:
here is the result:
```shell
1) test262 module tests
       Abrupt completion from module evaluation is reported (eval-self-abrupt.js) :

      AssertionError [ERR_ASSERTION]: Input A expected to strictly equal input B:
+ expected - actual

- 'ReferenceError'
+ 'Test262Error'
      + expected - actual

      -ReferenceError
      +Test262Error
```

Reply:
It's a reference error if `Test262Error ` doesn't exist.
<img width="375" src="https://user-images.githubusercontent.com/4303/43810508-ecb656ce-9a6c-11e8-93c3-28936d3b9077.png">

Reply:
uh, my bad. 😞
I thought the Test262 exceptions are always a string. guess not. fixing ...
thanks!

Reply:
verified ✅ 

this was some regression on my side. Test262Error was being injected (just not always, depending on the flag). I'll fix properly later, if needed.